autoload.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 $autoloadFile = './vendor/codeception/codeception/autoload.php';
-if (file_exists('./vendor/autoload.php') && file_exists($autoloadFile) && __FILE__ != realpath($autoloadFile)) {
+if (!isset($argv['--prefer-local']) && file_exists('./vendor/autoload.php') && file_exists($autoloadFile) && __FILE__ != realpath($autoloadFile)) {
```

Comment:
Does this condition even work? `$argv` isn't an associative array, it uses 0-based indexes.

Reply:
Please test this change in your own project, install Codeception from this branch and check if it works as you expect it to.
There is no point to merge it if it turns out that it doesn't solve your issue.

Reply:
I have changed this.